nubi/src/assets/images/logos.js

```diff
@@ -0,0 +1,23 @@
+import playstation from './img_logo_playstation.png';
+import forever21 from './img_logo_forever21.png';
+import twitch from './img_logo_twitch.png';
+import tripadvisor from './img_logo_tripadvisor.png';
+import airbnb from './img_logo_airbnb.png';
+import ebay from './img_logo_ebay.png';
+import tiendamia from './img_logo_tiendamia.png';
+import carters from './img_logo_carters.png';
+import origin from './img_logo_origin.png';
+
+const logos = [
+  { src: airbnb, title: '|' },
```

Comment:
Siempre, por accesibilidad, es mejor usar un title que refleje lo que tiene. Recomendaría algo como:
```suggestion
  { src: airbnb, title: 'Logo Airbnb' },
```
Y así para los que están abajo.

Reply:
tambien modifique los titles para que sean mas acorde a lo que muestran.

Reply:
💪